### PR TITLE
Fixed autocreate on nonPersistentInverseOwner

### PR DESCRIPTION
**Pull request by hAdamm**

### CODE REVIEW

File: src/Relationships/One.php [hunk 1]
```diff
@@ -90,18 +90,22 @@ public function entityForOwnerForeignKey(\obo\Entity $owner, array $foreignKey,
...
-            $specification->where("{{$foreignKey[0]}} = " . \obo\Interfaces\IQuerySpecification::PARAMETER_PLACEHOLDER . " AND {{$foreignKey[1]}} = " . \obo\Interfaces\IQuerySpecification::PARAMETER_PLACEHOLDER, $owner->primaryPropertyValue(), $owner->entityInformation()->name);
+        if ($owner->isBasedInRepository()) {
+            if (\count($foreignKey) === 1) {
+                $specification->where("{{$foreignKey[0]}} = " . \obo\Interfaces\IQuerySpecification::PARAMETER_PLACEHOLDER, $owner->primaryPropertyValue());
+            } else {
+                $this->ownerNameInProperty = $foreignKey[1];
+                $specification->where("{{$foreignKey[0]}} = " . \obo\Interfaces\IQuerySpecification::PARAMETER_PLACEHOLDER . " AND {{$foreignKey[1]}} = " . \obo\Interfaces\IQuerySpecification::PARAMETER_PLACEHOLDER, $owner->primaryPropertyValue(), $owner->entityInformation()->name);
+            }
+
+            if ($entity = $entityManagerName::findEntity($specification, false))  return $entity;
```
Comment (lordjim666) [inline src/Relationships/One.php]: 2 spaces

File: tests/TestCases/Entity/AutoCreateTest.phpt [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        $administrativeEmail = $this->createNonPersistentAdministrativeEmail();
+        \Tester\Assert::true($administrativeEmail->contact instanceof \obo\Tests\Assets\Entities\Contacts\Contact);
+    }
+
+    public function testReadOnPersistentInverseOwner() {
+        $administrativeEmail = $this->createPersistentAdministrativeEmail();
+        \Tester\Assert::true($administrativeEmail->contact instanceof \obo\Tests\Assets\Entities\Contacts\Contact);
+    }
+
+    public function testWriteOnNonPersistenInversetOwner() {
```
Comment (lordjim666) [inline tests/TestCases/Entity/AutoCreateTest.phpt]: inverset is not a word

File: tests/TestCases/Entity/__assets/Entities/Contacts/Contact/AdministrativeEmail/AdministrativeEmailProperties.php [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace obo\Tests\TestCases\Entity\Assets\Entities\Contacts\Contact;
+
+class AdministrativeEmailProperties extends \obo\Tests\Assets\Entities\Contacts\AdditionalInformation\EmailProperties {
+
+    /**
+     * @obo-one(targetEntity="\TestCases\Entity\Assets\Entities\Contacts\Contact", connectViaProperty = "administrativeEmail", autoCreate = true)
```
Comment (lordjim666) [inline tests/TestCases/Entity/__assets/Entities/Contacts/Contact/AdministrativeEmail/AdministrativeEmailProperties.php]: should be there obo... in namespace?

Comment (hAdamm) [inline tests/TestCases/Entity/__assets/Entities/Contacts/Contact/AdministrativeEmail/AdministrativeEmailProperties.php]: NO

**Pull request merged**